mad-dog/src/index.js

```diff
@@ -68,15 +69,16 @@ async function tearDownAllServices () {
 // This flag is set to 'true' to run the test with the 0th indexed wallet.
 // The default will be 'undefined' for the other tests that do not require
 // this flag.
-const makeTest = (name, testFn, { numUsers, numCreatorNodes, useZeroIndexedWallet }) => {
+const makeTest = (name, testFn, { numUsers, numCreatorNodes, useZeroIndexedWallet, ...restParams }) => {
```

Comment:
is the purpose of `restParams` for other configs like `iterations` that you specified in `userReplicaSetNodesTest`? if so/nit, i'd rename this `additionalConfigs`.